compose.yml

```diff
@@ -1,18 +1,19 @@
 services:
     frontend:
-        container_name: "frontend_${COMPOSE_PROJECT_NAME:-ts}"
+        container_name: 'frontend_${COMPOSE_PROJECT_NAME:-ts}'
```

Comment:
Appending the -ts, does that solve compiling across multiple versions of the project? I think you explained it before but not specifically what changed

Reply:
this adds a `-ts` suffix when the COMPOSE_PROJECT_NAME is not set (which it probably isnt.)
so if you run compose like this:
`COMPOSE_PROJECT_NAME=CHECKPR docker compose up --build`
it appends `CHECKPR` to all the containers to stop it from complaining about "container already in use"